config.toml.example

```diff
@@ -255,6 +255,13 @@ changelog-seen = 2
 # Defaults to the Python interpreter used to execute x.py
 #python = "python"
 
+# The path to the REUSE executable to use. REUSE will be used to gather
+# the licensing information of the codebase.
+#
+# If this is omitted, the cached licensing information present in the source
+# tarball will have to be present.
```

Comment:
```suggestion
# If this is omitted, the cached licensing information present in the source
# tarball will have to be present, and there we be no attempt to invoke
# a REUSE executable during the build process.
```

Reply:
This is actually incorrect, I wrote this before the rest of the implementation and I ended up going for a different approach than planned. I'll revise the comment.